test/external/external_model_benchmark.py

```diff
@@ -13,18 +13,13 @@
 MODELS = {
   "resnet50": "https://github.com/onnx/models/raw/main/validated/vision/classification/resnet/model/resnet50-caffe2-v1-9.onnx",
   "openpilot": "https://github.com/commaai/openpilot/raw/v0.9.4/selfdrive/modeld/models/supercombo.onnx",
+  "commavq": "https://huggingface.co/commaai/commavq-gpt2m/resolve/main/gpt2m.onnx",
   "efficientnet": "https://github.com/onnx/models/raw/main/validated/vision/classification/efficientnet-lite4/model/efficientnet-lite4-11.onnx",
   "shufflenet": "https://github.com/onnx/models/raw/main/validated/vision/classification/shufflenet/model/shufflenet-9.onnx",
-  "commavq": "https://huggingface.co/commaai/commavq-gpt2m/resolve/main/gpt2m.onnx",
-
-  # broken in torch MPS
-  # "zfnet": "https://github.com/onnx/models/raw/main/archive/vision/classification/zfnet-512/model/zfnet512-9.onnx",
-  # TypeError: BatchNormalization() got an unexpected keyword argument 'is_test'
-  # "densenet": "https://github.com/onnx/models/raw/main/archive/vision/classification/densenet-121/model/densenet-3.onnx",
-  # AssertionError: only onnx version >= 10 supported for slice
-  # "bert": "https://github.com/onnx/models/raw/main/archive/text/machine_comprehension/bert-squad/model/bertsquad-8.onnx",
-  # really slow
-  # "resnet18": "https://github.com/onnx/models/raw/main/archive/vision/classification/resnet/model/resnet18-v2-7.onnx",
+  "densenet": "https://github.com/onnx/models/raw/main/Computer_Vision/densenet121_Opset18_timm/densenet121_Opset18.onnx",
```

Comment:
does these work now? is it because some fix in this pr or if it always work?

Reply:
these worked for awhile now, like for `bert`, `slice` support for onnx version < 10 was implemented quite some time back, it's just that this wasn't updated.

Also some of the links were dead links. (url changed)

I guess I'll submit this change as a separate PR